vtg.summary/src/R/RPC_summary.R

```diff
@@ -0,0 +1,169 @@
+#' Get summary statistics for a single node
+#'
+#' This function calculates all the preliminairy statistics used in the summary
+#' algorithm. It calculates number of NA or missing data, length of each column
+#' in the data, sum per column of data given that the data is not a factor,
+#' and the range per column of the data:- if the column is a factor
+#' then it returns a table of values with a disclosure check, otherwise
+#' it simply returns the min and max values. Finally this also returns the
+#' number of useable rows in the data.
+#'
+#' @param data A data.frame supplied by the node
+#' @param columns List of column names to compute summary for.
+#' @param threshold Minimum count in any result before error message is returned
+#' because the result may be disclosive. Default is 5.
+#' @param types types of the columns
+#' @param subset_rules Rules to filter data with. Default is NULL.
+#' @param is_extend_data Whether to extend the data with the `is_extend_data`
+#' function. Default is TRUE.
+#'
+#' @return A list with the following results for each column: length, sum,
+#' range and number of rows with non-empty and empty values.
+#'
+#' @TODO check if works with single column
+RPC_summary <- function(data, columns, threshold = 5L, types = NULL,
```

Comment:
This `threshold` cannot be user defined.. Read it of a custom environment variable: `SUBSET_MINIMAL_NUM_OBSERVATIONS` or something similar

Reply:
Same goes for all other `RPC_` functions